OpenRA.Mods.Common/Traits/Render/WithWallSpriteBody.cs

```diff
@@ -97,20 +106,18 @@ public void Tick(Actor self)
 			adjacent = 0;
 			foreach (var a in adjacentActors)
 			{
-				var rb = a.TraitOrDefault<WithWallSpriteBody>();
-				if (rb == null || rb.wallInfo.Type != wallInfo.Type)
+				CVec facing;
+				var wc = a.TraitsImplementing<IWallConnector>().FirstOrDefault(b => b.IsTraitEnabled());
```

Comment:
This is an unnecessary lambda expression and the code usually uses `Where/FirstOrDefault/etc(Exts.IsTraitEnabled)`.
When I add the `Trait` base class, I plan on adding `public static bool IsEnabled` so that `Exts.IsTraitEnabled` is replaced by `Trait.IsEnabled`.